.travis.yml

```diff
@@ -1,36 +1,37 @@
 language: python
 env:
+  global:
+    - TF_VAR_build_number=${TRAVIS_JOB_NUMBER/./-}
+    - DOCKER_ARGS='-i -a stdout -a stderr -e TF_VAR_build_number'
+    - DOCKER_IMAGE=sehqlr/mantl:testing
   matrix:
-     - TERRAFORM_FILE=testing/aws.tf ANSIBLE_PLAYBOOK=sample.yml
-     - TERRAFORM_FILE=testing/do.tf  ANSIBLE_PLAYBOOK=sample.yml
-     - TERRAFORM_FILE=testing/gce.tf ANSIBLE_PLAYBOOK=sample.yml
-    # - TERRAFORM_FILE=testing/aws.tf ANSIBLE_PLAYBOOK=kubernetes.yml
-    # - TERRAFORM_FILE=testing/do.tf  ANSIBLE_PLAYBOOK=kubernetes.yml
-    # - TERRAFORM_FILE=testing/gce.tf ANSIBLE_PLAYBOOK=kubernetes.yml
+    - TERRAFORM_FILE=testing/aws DOCKER_SECRETS='-e AWS_SECRET_ACCESS_KEY -e AWS_ACCESS_KEY_ID'
+    - TERRAFORM_FILE=testing/do DOCKER_SECRETS='-e DIGITALOCEAN_TOKEN'
+    - TERRAFORM_FILE=testing/gce DOCKER_SECRETS='-e GOOGLE_CREDENTIALS'
 
 install: "pip install -r requirements.txt"
 
-before_script:
+before_script: 
+  - mv $TERRAFORM_FILE $TERRAFORM_FILE.tf
```

Comment:
Shouldn't it be something more like `mv testing/aws.tf ./aws.tf`? Don't those files have extensions? Shouldn't they be moved to the working directory?